v2/config/samples/network/v1beta/v1beta20180901_privatednszone.yaml

```
@@ -0,0 +1,11 @@
+apiVersion: network.azure.com/v1beta20180601
```

```suggestion
apiVersion: network.azure.com/v1beta20180901
```

I just noticed failed on your PR.. ​runtime.notRegisteredErr​error indicates about either either of gvk in the sample is not correct. This should fix the error.

---

Thanks. I've just merged the same fix into my local branch.